### ISSUE TITLE
Tabs missing a line indicator

### ISSUE THREAD

**Issue by Emyeh** [HymnalDreamTeam/Hymns-iOS#134]
Tabs is a missing a blue line underneath. The line makes it A11y friendly specifically for color blind users.

![IMG_7412](https://user-images.githubusercontent.com/26554065/81532051-a4b45000-9318-11ea-9a73-aba93a1de0bc.PNG)

@finben @skywalkerdude
